tools/wpt/tests/test_wpt.py

```diff
@@ -91,6 +91,14 @@ def test_files_changed(capsys):
     assert err == ""
 
 
+def test_files_changed_ignore(capsys):
+    from wpt.testfiles import exclude_ignored
+    files = ["resources/testharness.js", "resources/webidl2/index.js", "test/test.js"]
```

Comment:
Having webidl2.js show up in a test would be nice, since that was the thing that originally didn't work.

Reply:
That's not entirely trivial to test since it only affects `tests_affected` rather than `get_files_changed` and that's mostly about reading files to see what data they contain, so it would either need to be refactored or we'd need to find some commit with a webidl2 change to test, but there aren't many of those. Maybe land the upgrade and use that commit for the test.

Reply:
Would a previous commit that changes webidl2.js do the job? 960a3d21bab0293630da8919847f87f4af3a3198?